sys/net/application_layer/asymcute/asymcute.c

```diff
@@ -86,14 +86,14 @@ static size_t _len_set(uint8_t *buf, size_t len)
 static ssize_t _len_get(uint8_t *buf, size_t pkt_len, size_t *len)
 {
     if (buf[0] != 0x01) {
-        *len = (uint16_t)buf[0];
+        *len = (size_t)buf[0];
```

Comment:
```suggestion
        *len = buf[0];
```

automatic integer promotion should do the trick already afaik, but maybe someone else could confirm? @maribu @Teufelchen1